### ISSUE TITLE
The selected pivot doesn't announce that it's selected in Narrator

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1609]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. Expand arrow is unlabeled
Expect: Should say "Expand or collapse <item name"

2. Visible focus does not follow actual focus

**Comment by betrue-final-final** [comment]
In Narrator, it goes straight to the expand arrow, and doesn't read the node. Should read the node when you land in the Nav. Other bugs are not repro in Narrator.

**Comment by betrue-final-final** [comment]
Also, the selected pivot should announce it is selected.

**Comment by betrue-final-final** [comment]
Still reproes the second comment issue.
